example/rive_viewer.cpp

```diff
@@ -160,9 +171,51 @@ static void cleanExample()
     delete animationInstance;
 }
 
+static void animPopupItemCb(void *data EINA_UNUSED, Evas_Object *obj, void *event_info)
+{
+   int animationIndex = static_cast<int>(reinterpret_cast<intptr_t>(data));
+   initAnimation(animationIndex);
+   elm_ctxpopup_dismiss(statePopup);
+}
+
+static Elm_Object_Item* animPopupItemNew(Evas_Object *obj, const char *label, int index)
+{
+   Elm_Object_Item *it = nullptr;
+
+   if (!obj) return nullptr;
+
+   return elm_ctxpopup_item_append(obj, label, nullptr, animPopupItemCb, (void*)index);
+}
+
+static void animPopupDismissCb(void *data EINA_UNUSED, Evas_Object *obj, void *event_info EINA_UNUSED)
+{
+    if (statePopup)
+    { 
+      evas_object_del(obj);
```

Comment:
I think you don't need to compare if (statePopup)
otherwise,
just call evas_object_del(obj);